__tests__/plugin.test.ts

```diff
@@ -133,8 +88,51 @@ describe('babel plugin', () => {
     </script>`;
 
     const { code } = runPlugin(input);
-    expect(code).not.toContain('some comment');
-    expect(code).not.toContain('other comment');
+    expect(code).not.toMatch(/some comment/);
+    expect(code).not.toMatch(/other comment/);
```

Comment:
What was wrong with `toContain`? 😄

Reply:
when provided a regular expressions, toContain looks for regular expressions in the code, not strings that match the regexp